brave/brave-core
# Add warning about non-browser network context when use IPFS

Resolves https://github.com/brave/brave-browser/issues/23570
Warning is added in settings(when user chooses local node) and in onboarding screen(for local node option).

![image](https://user-images.githubusercontent.com/106654560/176511637-90f30df3-eeda-4ad6-a5d0-2755eb257e49.png)
![image](https://user-images.githubusercontent.com/106654560/176511730-ed2125ca-d009-490d-9461-97cc577e33be.png)
Also fixes displaying local node settings when local node is not selected.

Also we need to add some text here https://support.brave.com/hc/en-us/articles/360051406452-How-does-IPFS-Impact-my-Privacy-

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1) Open IPFS settings
2) Check that when local node is sellected, note is shown.
3) Select "ask" and load ipfs site.
4) Check that local node description contains note about proxy\dns settings.
5) Select non-localnode option and restart browser. Check that notice and other local node options are not displayed.
6) Select local node and restart browser. Check that notice and other local node options are displayed.
